config/services/kube-control-plane.xml

```diff
@@ -7,5 +7,4 @@
   <include service="kube-apiserver" />
   <include service="kube-controller-manager" />
   <include service="kube-scheduler" />
-  <include service="kube-api" />
```

Comment:
Dropping this may surprise users (kubelet may no longer be exposed). I think it's okay since the service was intended to be for the _control plane_. The presence of this here should be considered a bug.